airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -54,6 +54,8 @@ class ShopifyBulkManager:
     # to maximize the performance for multi-connection syncs and control the bulk job size within +- 1 hours (3600 sec),
     # Ideally the source will balance on it's own rate, based on the time taken to return the data for the slice.
     job_max_elapsed_time_sec: Final[float] = 1800.0
+    # retry max limit
```

Comment:
This comment may be unnecessary, as the identifier tells what the number is for. A better comment could explain why the 6. And maybe the identifier is too long. What about something like _job_error_max_retries.

Reply:
Corrected, thanks!

Reply:
6 - because the CDK `max_retries` default value is 6 as well.